src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx

```diff
@@ -21,170 +21,159 @@ import {
 } from '../config/helpers';
 import { benefitPhrases, veteranBenefits } from '../definitions/constants';
 
-export class ConfirmationPage extends React.Component {
-  componentDidMount() {
-    focusElement('h2');
+export const ConfirmationPage = props => {
+  useLayoutEffect(() => {
```

Comment:
- refactored this to a functional component because the timing of `useEffect`/`useLayoutEffect` is a little different than `componentDidMount`, which helped in this case.
- `useEffect` could work here too, but decided to use `useLayoutEffect`, because it is often used with scrolling.